### PR TITLE
Spanish Catalan and more taxonomy

### PR DESCRIPTION
**Pull request by AcuarioCat**
Additions and corrections

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -19271,7 +19287,7 @@ fr:tournesol oléique
 en:high oleic sunflower oil
 ca:oli refinat de gira-sol alt oleic
 de:High Oleic-Sonnenblumenöl
-es:aceite de girasol alto oleico,aceite refinado de girasol alto oleico,huile de tournesol haut oléique
+es:aceite de girasol alto oleico,aceite refinado de girasol alto oleico,huile de tournesol haut oléique,omega 9
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:19290]: why "omega 9" ?
also "huile de tournesol haut oléique" should be in the fr: after (but it's from another change)

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -71687,6 +71752,9 @@ wikidata:en:Q658413
 wikipedia:en:https://en.wikipedia.org/wiki/Bread_crumbs
 # ingredient/fr:panure has 462 products in 6 languages @2019-05-17
 
+<en:breadcrumbs
+en:wholmeal breadcrumbs
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:71756]: wholemeal ?

Comment (AcuarioCat) [reply taxonomies/ingredients.txt:71756]: Typo.. fixed

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt:19290]: Omega 9 from these 3:
https://es.openfoodfacts.org/ingrediente/omega-9

Comment (stephanegigandet) [reply taxonomies/ingredients.txt]: "Aceite de glrasol con alto contenido de acido oleico (omega 9),"
--> we should not make "Omega 9" a synonym of "aceite de girasol alto oleico". There are lots of other possible sources of Omega 9.

Instead we can add Omega 9 as a single ingredient. (without linking it to the different oils).

Comment (AcuarioCat) [reply taxonomies/ingredients.txt]: ok

**Pull request merged**